fractal_server/app/routes/auth/current_user.py

```diff
@@ -62,3 +65,36 @@ async def patch_current_user(
         patched_user, db
     )
     return patched_user_with_groups
+
+
+@router_current_user.get(
+    "/current-user/settings/", response_model=UserSettingsReadStrict
+)
+async def get_current_user_settings(
+    current_user: UserOAuth = Depends(current_active_user),
+    db: AsyncSession = Depends(get_async_db),
+) -> UserSettingsReadStrict:
+    user_settings = await db.get(UserSettings, current_user.user_settings_id)
```

Comment:
if `current_user.user_settings_id` is nullable, we should handling the a `None` from the get with something like:
```python
if not user.user_settings_id is None:
    raise HTTPException(status_code=404, detail="User settings not found")
```

Reply:
Thanks!
Done (also the ones below)